sotodlib/io/bookbinder.py

```diff
@@ -330,6 +332,10 @@ def preprocess(self):
             if self.nchans is None:
                 self.nchans = len(self.readout_ids)
                 self.primary_names = frame['primary'].names
+                for i, n in enumerate(self.primary_names):
+                    if n == "FrameCounter":
+                        fc_idx = i
+                        break
```

Comment:
Does `fc_idx = list(self.primary_names).index('FrameCounter')` not work?

Reply:
yep that works